semgrep-core/src/parsing/pfff/Python_to_generic.ml

```diff
@@ -735,6 +735,13 @@ and stmt_aux env x =
   | AugAssign (v1, (v2, tok), v3) ->
       let v1 = expr env v1 and v2 = operator v2 and v3 = expr env v3 in
       [ G.exprstmt (G.AssignOp (v1, (v2, tok), v3) |> G.e) ]
+  | Cast (Name (id, _kind, _ref), _tok, ty) ->
+      (* no need to guard with assign_to_vardef here *)
+      let id = name env id in
+      let ty = type_ env ty in
+      let ent = G.basic_entity id in
+      let def = G.VarDef { G.vtype = Some ty; vinit = None } in
```

Comment:
May be worth adding a comment that this is translated in this very specific way for DeepSemgrep's consumption?

Reply:
Well, after a second thought, I think this translation makes more sense in general anyways. What feels weird is that we parse `fld1 : int` as a cast in the first place. 🤷‍♂️